### PR TITLE
storage: no difference in running/installed packages

### LINKED ISSUE #398: Drop ReportUnknownPackage.running_*
Obsolete & unused.

### PR DESCRIPTION
**Pull request by sorki**
As we don't differentiate between running and installed packages this commit
- drops `ReportUnknownPackage.running_*`
- drops `installed_` prefix from used fields

Closes #398.

Signed-off-by: Richard Marko <rmarko@fedoraproject.org>

### CODE REVIEW

File: src/pyfaf/storage/migrations/versions/1c7edfbf8941_drop_reportunknownpackage_running_fields.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    try:
+        op.drop_constraint("reportunknownpackages_report_id_key", t)
+    except:
+        op.execute('ROLLBACK')
+        op.drop_constraint(
+            "reportunknownpackages_report_id_type_name_installed_epoch_i_key",
+            t)
+
+    op.drop_constraint("reportunknownpackages_installed_arch_id_fkey", t)
+    #op.drop_constraint("reportunknownpackages_arch_id_fkey", t)
```
Comment (mbrysa) [inline src/pyfaf/storage/migrations/versions/1c7edfbf8941_drop_reportunknownpackage_running_fields.py]: A leftover probably?

**Comment by mbrysa** [comment]
Have you tested it on RHEL 6?

**Comment by sorki** [comment]
Updated. It works on RHEL 6. I've added +1 commit adding alembic.ini to package.

**Comment by mbrysa** [comment]
Thank you!

**Pull request merged**